ext/random/random.stub.php

```diff
@@ -148,6 +148,7 @@ public function shuffleArray(array $array): array {}
 
         public function shuffleBytes(string $bytes): string {}
 
+        /** @return array<int, string> */
         public function pickArrayKeys(array $array, int $num): array {}
```

Comment:
This is not correct, the returned array keys might be integers. The correct type would be `list<int|string>`.